docs/types.rst

```diff
@@ -13,6 +13,8 @@ Solidity provides several elementary types which can be combined to form complex
 In addition, types can interact with each other in expressions containing
 operators. For a quick reference of the various operators, see :ref:`order`.
 
+The concept of "undefined" or "null" values do not exist in Solidity. To handle any unexpected values, you should use the :ref:`revert function<assert-and-require>` to revert the whole transaction.
+
```

Comment:
Something we should perhaps also mention here: Freshly declared Solidity variables always have a default value - there is no undefined behaviour. The default value depends on the type, but is usually straightforward (zero, false, empty, etc).

Reply:
@chriseth Added details to where reader can find more details on default values.